.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2016"]
```

Comment:
I think presets `env` is better and the recommended practice as of now. 

Check https://babeljs.io/env/